sechub-wrapper-xray/src/test/java/com/mercedesbenz/sechub/wrapper/xray/report/XrayWrapperReportParserTest.java

```diff
@@ -32,11 +33,11 @@ void getRootDataNode_throws_xrayWrapperReportException() {
         /* prepare */
         File file = new File("src/test/resources/invalid-json-examples/invalid_json.json");
         /* execute + test */
-        Assert.assertThrows(XrayWrapperReportException.class, () -> xrayWrapperReportParser.getRootDataNode(file));
+        Assert.assertThrows(XrayWrapperException.class, () -> xrayWrapperReportParser.getRootDataNode(file));
```

Comment:
please do an static import for "Assert.*" and use 
```suggestion
        assertThrows(XrayWrapperException.class, () -> xrayWrapperReportParser.getRootDataNode(file));
```